mlrun/render.py

```diff
@@ -96,20 +99,33 @@ def link_html(text, link=""):
 
 
 def artifacts_html(x, pathcol="path"):
-    if not x:
+    artifact_dicts, output_path = x
```

Comment:
Can we give `x` a more informative name?
I know it's called from the lambda function but still

Reply:
also type hinting would be useful, as well as changing the `pathcol` maybe to a more indicative name such as `attribute_name` ( just an example open for other naming )

Reply:
👍